web/pandas/about/team.md

```diff
@@ -36,6 +36,14 @@ If you want to support pandas development, you can find information in the [dona
     {% endfor %}
 </div>
 
+## Diversity and Inclusion
+
+> _pandas_ expressly welcomes and encourages contributions from anyone who faces under-representation, discrimination in the technology industry
+> or anyone willing to increase the diversity of our team.
+> We have identified the visible diversity gap and obstacles in the open source community and we are proactive in increasing the diversity of our team.
+> We have a [code of conduct]({base_url}/community/coc.html) to ensure a friendly and welcoming environment.
+> Please send an email to [mailto:pandas-code-of-conduct-committee](pandas-coc@googlegroups.com), if you don't feel this way.
```

Comment:
", if you don't feel this way"  doesn't really flow well with me. not sure if anyone else gets the same impression. if that's the case, my suggestion for rephrasing would be " Please send an email to [mailto:pandas-code-of-conduct-committee](pandas-coc@googlegroups.com) if you think we can do a better job at achieving this goal."